test/base/ViewValidator.scala

```diff
@@ -73,17 +32,6 @@ trait ViewValidator extends MustMatchers with ViewMatchers {
     errorLink.attr("href") must be(href)
   }
 
-  @deprecated("Please use 'page must haveFieldErrorLink(...)'", since = "2019-08-07")
```

Comment:
Do we have methods like `haveFieldErrorLink` defined already or is it part of the future works?

Reply:
Yes we have but in different trait - `ViewMatchers`